Dev/MetaPackageManager/meta_package_manager.7h.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 # <bitbar.title>Meta Package Manager</bitbar.title>
-# <bitbar.version>v2.5.0</bitbar.version>
+# <bitbar.version>v3.4.0</bitbar.version>
 # <bitbar.author>Kevin Deldycke</bitbar.author>
 # <bitbar.author.github>kdeldycke</bitbar.author.github>
 # <bitbar.desc>List outdated packages and manage upgrades.</bitbar.desc>
```

Comment:
Any chance you could add an image too?

Reply:
There's already an image! :) See: https://github.com/matryer/bitbar-plugins/pull/1479/commits/501efb61873bbc8bed071dd612c162e934091bd5#diff-35b8b72745b2f4b47516a8ceb1054c36f61ba6cbe877bd02a70af9b08f835aaeR9

Reply:
Image worth 1000 words:

![Screen Shot 2020-12-11 at 15 57 03](https://user-images.githubusercontent.com/159718/101918387-989fb200-3bc9-11eb-8f38-eb43a4c2f23e.png)

😉